src/components/v5/common/ActionSidebar/partials/AmountField/AmountField.tsx

```diff
@@ -65,7 +65,7 @@ const AmountField: FC<AmountFieldProps> = ({
     adjustInputWidth,
   } = useAmountField(tokenAddressController.value, maxWidth);
   const [value, setValue] = useState<string | undefined>(
```

Comment:
Maybe we can also remove the `undefined` from the useState type?

Reply:
+1 to this!

Reply:
Good pickup, thank you. Removed as well.